tests/assets/ansible/roles/node_base/vars/default.yaml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Why do we need that empty file?

Reply:
If there is no match for the include_vars this is the one included (though it most likely won't work obviously because no repos are defined)

It is not possible to ignore_errors in the lookup for `include_vars` like we did in `include_tasks`. We can ignore the task error itself though.

Do we prefer a direct failure ? I was not sure actually.

Reply:
Would `https://docs.ansible.com/ansible/latest/collections/ansible/builtin/first_found_lookup.html#parameter-skip` help here?

Reply:
I'll give it a try

Reply:
I just tried and skip is not allowed in include_vars lookup.
I think we can leave it like that.

`fatal: [rookcheck-bleon-9ff3-worker-2]: FAILED! => {"msg": "{{ lookup('first_found', params, skip is not a valid option in include_vars"}`